src/search/BracketsSearchCursor.js

```diff
@@ -350,6 +350,29 @@ define(function (require, exports, module) {
         }
         return true;
     };
+    /**
+     * Determines if the groupNum1 of this Array is a enclosed by groupNum2 of groupArray2
+     * @private
+     * @param   {GroupArray} groupNum1   Item within group array
+     * @param   {GroupArray} groupArray2 Second group array
+     * @param   {GroupArray} groupNum2   Item within groupArray2
+     * @return  {boolean}  true when the groupNum1 of this Array is a enclosed by groupNum2 of groupArray2
+     */
+    GroupArray.prototype.isGroupEnclosed = function (groupNum1, groupArray2, groupNum2) {
```

Comment:
Since this is no longer generic, I'm thinking maybe it shouldn't be part of the GroupArray.
Maybe it should be a function named something like 'isResultsOverlapping' as it is checking specific values in the group.